### PR TITLE
Courtney/feature/login page

### LINKED ISSUE #9: Feature: User can log in
As a user, when I visit the login page, I want to put in a username and password so I can log in to the application using my credentials. 

Image

Subtasks:

- [x] Create a login component (should contain username and password fields)
- [x] Create a Route for the login page
- [x] User should be redirected to the homepage after successful login
- [x] If an incorrect user or password is typed, the user should see a message Oops, we cannot find that username/password. Please try again.

### PR DESCRIPTION
**Pull request by Courtney-Lippman**
- [ ]  Wrote Tests 
- [x] Implemented 
- [ ] Reviewed

## Necessary checkmarks:

- [x] All Tests are Passing in all environments

- [x] The code will run locally

## Type of change

- [x] New feature
- [ ] Bug Fix
- [x] Refactor
- [x] Styling
- [ ] Testing
- [ ] This change requires a documentation update

## Description of Change:
This PR will add functionality to the Login Page so that when a user inputs a correct username and password and clicks the login button, they will be redirected to their home page with their feed. If the user does not input a valid password/username and clicks the login button, an invalid username/password message with appear, and the username/password inputs will be cleared.

correct username: Hank

universal correct password: streamr

## Check the correct boxes

- [x]  This broke nothing
- [ ]  This broke some stuff
- [ ]  This broke everything

## Checklist:

- [x]  My code has no unused/commented out code
- [x]  My code follows the style guidelines of this project
- [x]  My code does not generate new warnings
- [x]  I have reviewed my code
- [ ]  I have commented my code, particularly in hard-to-understand areas (if applicable)

### CODE REVIEW

File: src/Components/LoginPage/_LoginPage.scss [hunk 1]
```diff
@@ -66,5 +66,10 @@ $text-color: #D9D9D9;
             }
 
         }
+
+        p {
+            font-size: 1.75vw;
+            color: #44c4a1;
```
Comment (ghost) [inline src/Components/LoginPage/_LoginPage.scss]: I see color names at the top being used, would recommend also adding this as a color for reference (Ex. $login-area-text etc.)

File: src/Components/LoginPage/LoginPage.js [hunk 2]
```diff
@@ -1,12 +1,54 @@
...
+        loggedIn: true
+      }))
+
+    } else {
+      setSignInData((prevState) => ({
+        ...prevState,
+        validSignIn: false,
+        loggedIn: false
+      }))
+      clearLogin()
```
Comment (ghost) [inline src/Components/LoginPage/LoginPage.js:41]: Curious why setting and clearing back to back?

File: src/Components/LoginPage/LoginPage.js [hunk 3]
```diff
@@ -21,20 +63,26 @@ const LoginPage = () => {
...
           type='password'
           placeholder='Password'
           name='password'
-          // value={password}
-          // onChange={handleChange}
+          value={signInData.password}
+          onChange={handleChange}
           className="input"
         />
         <button onClick={event => handleSubmit(event)}>Login</button>
```
Comment (ghost) [inline src/Components/LoginPage/LoginPage.js:78]: What happens if there's nothing typed or a space typed and you click login?

**Comment by ghost** [comment]
I would recommend adding screenshots to see what is getting updated and what it should look. Super useful for FE to stick to reviewing code without pulling the branch.

File: src/Components/LoginPage/LoginPage.js (hunk 3 above)
Comment (Courtney-Lippman) [reply src/Components/LoginPage/LoginPage.js:78]: If nothing is typed or space is typed, and the user clicks login, the user will see a message that says that the username/password was incorrect and the inputs are reset. The onChange updates the value as the space or an empty string if the user does not put anything in the input field. The handleSubmit checks to see if the values are valid usernames and passwords.

File: src/Components/LoginPage/_LoginPage.scss (hunk 1 above)
Comment (Courtney-Lippman) [reply src/Components/LoginPage/_LoginPage.scss]: That is a great catch! I will change the color to be assigned to a variable like the variables for the colors at the top of this file!

**Comment by Courtney-Lippman** [comment]
I created a color name for the color on line 72 in _LoginPage.scss. I also added clarifying comments to the comments that you made. @hemeshvpatelHE Let me know if my answers answered your comments/ questions. Thank you for taking the time to review this PR!

**Review by ghost — APPROVED** [review]

**Pull request merged**